tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingTestCase.java

```diff
@@ -134,22 +135,23 @@ public void testPull() throws Exception {
     @Test(description = "Test searching a package from central", dependsOnMethods = "testPush")
     public void testSearch() throws BallerinaTestException, IOException {
         String[] clientArgs = {moduleName};
-        String msg = "Ballerina Central\n" +
-                "=================\n" +
-                "\n" +
-                "|NAME                                                  | DESCRIPTION                                " +
-                "                                       | AUTHOR         | DATE           | VERSION |\n" +
-                "|------------------------------------------------------| -------------------------------------------" +
-                "---------------------------------------| ---------------| ---------------| --------|\n" +
-                "|" + orgName + "/" + moduleName + "                             | Prints \"hello world\" to " +
-                "command line output" +
-                "                                       |                | " + datePushed + " | 0.0.1   |\n";
-
-        LogLeecher clientLeecher = new LogLeecher(msg);
-        balClient.runMain("search", clientArgs, envVariables, new String[]{},
-                new LogLeecher[]{clientLeecher}, balServer.getServerHome());
-
-        clientLeecher.waitForText(3000);
+        LogLeecher clientLeecherOne = new LogLeecher("Ballerina Central");
+        LogLeecher clientLeecherTwo = new LogLeecher("=================");
+        LogLeecher clientLeecherThree = new LogLeecher("|NAME            | DESCRIPTION                   | DATE     " +
+                                                               "      | VERSION |");
+        LogLeecher clientLeecherFour = new LogLeecher("|----------------| ------------------------------| " +
+                                                              "---------------| --------|");
+        LogLeecher clientLeecherFive = new LogLeecher("|integrationte...| Prints \"hello world\" to com...| " +
+                                                           datePushed + " | 0.0.1   |");
+        balClient.runMain("search", clientArgs, envVariables, new String[]{}, new LogLeecher[]{clientLeecherOne,
+                                  clientLeecherTwo, clientLeecherThree, clientLeecherFour, clientLeecherFive},
+                          balServer.getServerHome());
+
+        clientLeecherOne.waitForText(3000);
+        clientLeecherTwo.waitForText(3000);
+        clientLeecherThree.waitForText(3000);
+        clientLeecherFour.waitForText(3000);
+        clientLeecherFive.waitForText(3000);
```

Comment:
Since the log leecher timeout is the maximum time we will wait for a log and the log leecher stops waiting as soon as the log is found, IMO this will not be an issue. I used the same timeout as the original tests assuming that it accounted for possible delays, but we could reduce it if we are sure it wouldn't take that long.

WDYT?

Reply:
Yeah we can reduce the maximum time to be 1 second